modules/aws/bootstrap/README.md

```diff
@@ -0,0 +1,51 @@
+# Bootstrap Module
```

Comment:
can you combine this with 5c536a24ce6860a32b9ba02eeef8d0a38ecc39ba ?

Reply:
> can you combine this with 5c536a2 ?

Yes.  I made 2ac5913 a separate commit so I could revert it once openshift/release#1677 lands.  But I can squash them if you like.  Thoughts?